runtime/bcutil/cfreader.c

```diff
@@ -1759,14 +1759,14 @@ checkMethods(J9PortLibrary* portLib, J9CfrClassFile* classfile, U_8* segment, U_
 			 * additionally have its ACC_STATIC flag (Section 4.6 Methods) set in order to be the class or interface
 			 * initialization method.
 			 */
-			if (classfileVersion < BCT_Java7MajorVersionShifted) {
+			if (classfileVersion < BCT_JavaMajorVersionShifted(7)) {
 				method->accessFlags |= CFR_ACC_STATIC;
 
 			/* Leave this here to find usages of the following check:
 			 * J2SE_19 has been deprecated and replaced with J2SE_V11
 			 * if (J2SE_VERSION(vm) >= J2SE_V11) { 
 			 */
```

Comment:
Please remove this now obsolete (and incorrect) comment.

Reply:
Removed.